website/docs/feature-flag-tutorials/use-cases/scaling-unleash.mdx

```diff
@@ -0,0 +1,323 @@
+
+
+<details>
+  <summary>Configuring self-hosted Unleash Edge for production</summary>
+
+    For a standard production setup of self-hosted Unleash Edge, we recommend the following:
+
+    - Run in Edge mode: This mode ensures the Edge instance connects to your Unleash API server to synchronize feature flags, segments, and required API tokens.
+    - Use initialization tokens: Configure Edge to start with initialization tokens, as this enables Edge to immediately pre-fetch flag configurations upon startup. This ensures low-latency responses to initial SDK requests, particularly when multiple Edge instances are load-balanced and serving frontend API traffic.
+    - Implement health and readiness checks: Integrate the built-in `/health` and `/ready` endpoints with your load balancer and orchestration platform. This ensures that traffic is only directed to healthy Edge instances that have completed their initial configuration loading.
+
+      For more information,refer to the Unleash Edge [deployment guide](/reference/unleash-edge/deploying).
+</details>
+
+### Lower scale or single-region scenarios
+
+Even when global scaling and minimizing cross-region latency are not immediate priorities, implementing high availability is crucial for reliable production workloads using Unleash.
+
+A standard approach involves addressing potential single points of failure within a region:
+
+- **Deploy redundant API instances**: Run multiple Unleash API instances across different Availability Zones (AZs). Place these instances behind a load balancer. The load balancer distributes traffic and routes requests away from failed instances or AZs, maintaining application-level availability.
+- **Use a multi-AZ database**: Configure your managed database service (such as AWS RDS for PostgreSQL) for multi-AZ deployment. This provides data resilience and automatic failover for the database layer.
+- **Configure your SDKs**: Set [SDKs](/reference/sdks) to connect to the Unleash server via the load balancer endpoint. Consider deploying highly-available Unleash Edge instances as intermediaries. Edge caches flags locally, improving performance and resilience.
+
+This configuration mitigates single points of failure for the Unleash API server and its database within a region.
+
+## Monitoring and observability
+
+Unleash Cloud provides built-in observability dashboards, usage metrics, and system status updates via the Admin UI and a [public status page](https://unleash.instatus.com/).
+
+Underlying infrastructure monitoring, alerting, and scaling are handled by Unleash SREs.
+
+Use the [Connected Edges dashboard](/reference/network-view#connected-edges) to get observability metrics for Unleash Edge instances, such as ID, region, resource usage, and latency.
+
+<details>
+  <summary>Self-hosted observability</summary>
+
+    - **Health checks**: Poll `/health` endpoints on API and Edge instances. Monitor `/internal-backstage/prometheus` on Edge.
```

Comment:
/internal-backstage/prometheus works also for Unleash, could be a good recommendation here (unless this section is just for edge)